docs/resources/global_mode.md

```diff
@@ -27,6 +27,6 @@ resource "wallarm_global_mode" "global_block" {
 
 ## Argument Reference
 
-* `waf_mode` - (Optional) WAF mode. Possible values: `default`, `monitoring`, `block`
-* `scanner_mode` - (Optional) Scanner mode. Possible values: `off`, `on`
-* `rechecker_mode` - (Optional) Attack rechecker mode. Possible values: `off`, `on`
+* `waf_mode` - (optional) global filtering mode. Possible values: `default`, `monitoring`, `block`
+* `scanner_mode` - (optional) Scanner mode. Possible values: `off`, `on`
+* `rechecker_mode` - (optional) Attack rechecker mode. Possible values: `off`, `on`
```

Comment:
what are default values?

Reply:
Default value for scanner_mode is "on". Default value for rechecker_mode is "off".